en/topic-categories.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Topics
+permalink: /en/topic-categories/
```

Comment:
Would this be better as `/en/topics-categories/`? Same for `topics-dates`?

Reply:
Hmm.  I think of it as a phrase, "topic categories".  You seem to be thinking of it as a classification, "topics→categories".  I don't have a strong preference, so I'm willing to change it if you want.

Reply:
No strong preference. Keep it as it is if you prefer it.